### PR TITLE
docs(sumologicexporter): refactor and update sumologicexporter README

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: pkg/exporter/sumologicexporter/README.md [hunk 1]
```diff
@@ -4,37 +4,49 @@ This exporter supports sending logs and metrics data to [Sumo Logic](https://www
...
 
-- `endpoint` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
-- `compress_encoding` (optional): Compression encoding format, either empty string (`""`), `gzip` or `deflate` (default `gzip`).
-Empty string means no compression
-- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default `1_048_576` (1MB) is used.
+- `endpoint` (required, unless configured with [`sumologicextension`][sumologicextension]): Unique URL generated for your HTTP Source.
+  This is the address to send metrics to.
+- `compress_encoding` (optional): Compression encoding format. Empty string means no compression
+  - default: `gzip`
+  - allowed values: `gzip`, `deflate`, `""`
```
Comment (sumo-drosiek) [inline pkg/exporter/sumologicexporter/README.md:11]: IMHO this indicates, that `default` and `allowed_values` is subconfiguration for `compress_encoding`.(see `queue_size` for example). Could we mark `default` and `allowed values` without bullet points?

Comment (pmalek-sumo) [reply pkg/exporter/sumologicexporter/README.md:11]: Yeah... I was thinking about that but then when it's all in one line it's all a huge pile of information and it's hard to find a particular setting/property that you're looking for (e.g. if you want to find the default value you have to peruse the whole bullet point entry to find it :/ )

Comment (sumo-drosiek) [reply pkg/exporter/sumologicexporter/README.md:11]: We could put as separate line in this bullet point

list:
  - `compress_encoding` (optional): Compression encoding format. Empty string means no compression
    default: `gzip`
    allowed values: `gzip`, `deflate`, `""`

```
list:
  - `compress_encoding` (optional): Compression encoding format. Empty string means no compression
    default: `gzip`
    allowed values: `gzip`, `deflate`, `""`
```

File: pkg/exporter/sumologicexporter/README.md [hunk 2]
```diff
@@ -2,43 +2,94 @@
...
+    source_category: <source_category>
+    # desired source name, useful if you want to override the source name
+    # configured for the source.
+    source_name: <source_name>
+    # desired host name, useful if you want to override the source host
+    # configured for the source.
+    source_host: <source_host>
+    # template for Graphite format, applied only if metric_format is set to graphite;
+    # source templating is going to be applied,
+    # default=`%{_metric_}`
```
Comment (sumo-drosiek) [inline pkg/exporter/sumologicexporter/README.md:46]:
```suggestion
    # default = `%{_metric_}`
```

**Comment by sumo-drosiek** [comment]
One minor comment from me :rocket:

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**